src/Illuminate/Database/Eloquent/SoftDeletes.php

```diff
@@ -38,7 +38,7 @@ public function forceDelete()
     /**
      * Perform the actual delete query on this model instance.
      *
-     * @return void
+     * @return mixed
```

Comment:
It returns count of affected rows if forced deleting or void if soft. I think it's better to change logic to always return `int` and leave this `mixed` until then. Your choice.